hack/build/format.sh

```diff
@@ -23,3 +23,4 @@ source "${script_dir}"/config.sh
 shfmt -i 4 -w ${CDI_DIR}/hack ${CLONER_MAIN}
 goimports -w -local kubevirt.io ${CDI_DIR}/cmd/ ${CDI_DIR}/pkg/ ${CDI_DIR}/test/
 (cd ${CDI_DIR} && go vet $(go list ./... | grep -v -E "vendor|pkg/client" | sort -u))
+gofmt -l -s -w $(SOURCE_DIRS)
```

Comment:
`goimports` wraps `gofmt` and add import sorting. I think the result here would be that L26 never catches format issues since L24 will have fixed them already.